src/test/scala/com/microsoft/hyperspace/index/CreateIndexTests.scala

```diff
@@ -87,6 +85,27 @@ class CreateIndexTests extends SparkFunSuite with SparkInvolvedSuite {
     assert(exception.getMessage.contains("Index config is not applicable to dataframe schema"))
   }
 
+  test("Index creation passes with columns of different case if case-sensitivity is false.") {
+    hyperspace.createIndex(df, IndexConfig("index1", Seq("qUeRy"), Seq("ImpRS")))
+    val indexes = hyperspace.indexes.where(s"""name = "${indexConfig1.indexName}" """)
+    assert(indexes.count == 1)
+    assert(
+      indexes.head.getAs[WrappedArray[String]]("indexedColumns").head.equals("Query"),
+      "Indexed columns with wrong case are stored in metadata")
+    assert(
+      indexes.head.getAs[WrappedArray[String]]("includedColumns").head.equals("imprs"),
+      "Included columns with wrong case are stored in metadata")
+  }
+
+  test("Index creation fails with columns of different case if case-sensitivity is true.") {
+    withSparkConf("spark.sql.caseSensitive", true) {
```

Comment:
Can we use withSQLConf? https://github.com/microsoft/hyperspace/blob/48b3332bf5aaca0c2daf890c9156113b62a65843/src/test/scala/com/microsoft/hyperspace/index/rules/JoinIndexRuleTest.scala#L115

Reply:
thanks @imback82 , fixed